ocrd_anybaseocr/cli/ocrd_anybaseocr_cropping.py

```diff
@@ -65,11 +65,11 @@ def write_crop_coordinate(self, base, coordinate):
     def remove_rular(self, arg):
         #base = arg.split(".")[0]
         #img = cv2.cvtColor(arg, cv2.COLOR_RGB2BGR)
-        #gray = cv2.cvtColor(arg, cv2.COLOR_BGR2GRAY)
+        gray = cv2.cvtColor(arg, cv2.COLOR_BGR2GRAY)
```

Comment:
@kba and @wrznr , can you please tell me if this process work for a local image file? 
Because, it is possible that the cropping script could receive a binarized image as an input.